.github/workflows/inter-branch-merge-flow.yml

```diff
@@ -0,0 +1,13 @@
+name: Inter-branch merge workflow
+on:
+  push:
+    branches:
+      - release/**
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  Merge:
+    uses: dotnet/arcade/.github/workflows/inter-branch-merge-base.yml@main
```

Comment:
Is it normal to use `main` in a release branch?  I would expect each release branch to use the respective release branch of arcade (and ideally at a matching commit to that arcade which is ingested in the repo).  @mmitche

Reply:
We use main in arcade for publishing and other common automation (including this automation in its current form). The scripting for inter-branch merging has changed minimally over the years and isn't release specific in any way.